### DISCUSSION TITLE
typo error in 2 questions

### DISCUSSION THREAD

**Discussion by Zayd-R** [spamegg1/Epp-Discrete-Math-5th-solutions#5] [General]
page 93, exercise set 4.6, question 23, 24 

when declaring the definition of floor : 

 n < x < n - 1
should be n < x < n + 1

**Comment by spamegg1** [comment]
That's right, thanks for the catch! Fixed [here](https://github.com/spamegg1/Epp-Discrete-Math-5th-solutions/commit/a6a4cd5540f15eeac3b0e61de6f24c79f95fef19)
